platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/schedulers/builders/TaskSchedulerBuilder.java

```diff
@@ -243,6 +243,7 @@ private record Counters(@NonNull ObjectCounter onRamp, @NonNull ObjectCounter of
      * @param outerCounter the counter provided by the outer scope, or null if not provided
      * @return the combined counter, or a no op counter if both are null
      */
+    @SuppressWarnings("java:S6416")
```

Comment:
Can you please add a comment on which warning we are suppressing here?

Reply:
Just spoke with @cody-littley. He proposed we will not add any suppress on that class just mark it as not fix on the Sonar side.